src/main/java/com/metamx/collections/bitmap/WrappedRoaringBitmap.java

```diff
@@ -146,7 +147,6 @@ public void serialize(ByteBuffer buffer)
     if (compressRunOnSerialization) {
       bitmap.runOptimize();
     }
-    buffer.putInt(getSizeInBytes());
```

Comment:
this method was never used in druid, so it wont break anything in druid atleast.